src/olympia/discovery/data.py

```diff
@@ -11,47 +11,39 @@ def __init__(self, *args, **kwargs):
 
 
 # At the moment the disco pane items are hardcoded in this file in the repos,
-# which allows us to integrate in our translation workflow easily.
+# which allows us to integrate in our translation workflow easily. Add-on ids
+# are used instead of slugs to prevent any accidental replacement of a deleted
+# add-on by another.
 discopane_items = [
-    # Theme: Symphony of colors
-    DiscoItem(addon_id=628864, addon_name=u'Symphony of Colors'),
 
-    # uBlock Origin
+    # 'Books' theme.
+    DiscoItem(addon_id=778525),
+
+    # Video DownloadHelper
     DiscoItem(
-        addon_id=607454,
-        heading=_(u'Block ads {start_sub_heading}with {addon_name}'
+        addon_id=3006,
+        heading=_(u'Download videos {start_sub_heading}with {addon_name}'
                   u'{end_sub_heading}'),
         description=string_concat(
             '<blockquote>',
-            _(u'A lightweight and effective ad blocker. uBlock Origin '
-              u'enforces thousands of content filters without chewing up a '
-              u'bunch of memory.'),
+            _(u'Easily grab videos from YouTube, Dailymotion, Facebook, and '
+              u'most other video sites with this simple download tool.'),
             '</blockquote>')),
 
-    # Emoji Keyboard
+    # uBlock Origin
     DiscoItem(
-        addon_id=674732,
-        heading=_(u'Emoji expression {start_sub_heading}with {addon_name}'
+        addon_id=607454,
+        heading=_(u'Block ads {start_sub_heading}with {addon_name}'
                   u'{end_sub_heading}'),
         description=string_concat(
             '<blockquote>',
-            _(u'Dozens of amazing emojis for every occasion—always just one '
-              u'click away.'),
+            _(u'A lightweight and effective ad blocker. uBlock Origin '
+              u'enforces thousands of content filters without chewing up a '
+              u'bunch of memory.'),
             '</blockquote>')),
 
-    # Theme: Stained Glass Fractal
-    DiscoItem(addon_id=465609),
-
-    # OmniSidebar
-    DiscoItem(
-        addon_id=296534,
-        heading=_(u'Easily access bookmarks {start_sub_heading}with '
-                  u'{addon_name}{end_sub_heading}'),
-        description=string_concat(
-            '<blockquote>',
-            _(u'Are you constantly scrolling through your bookmarks? '
-              u'Bring your lists into view with a single, simple gesture.'),
-            '</blockquote>')),
+    # 'yellowfield 1' theme.
+    DiscoItem(addon_id=599252, addon_name='Yellow Field 1'),
```

Comment:
Hmm, the copy is kinda unclear on this. On the one hand it calls this just `Yellowfield` and in the ordering at the bottom it says `Yellow field 1` - I'm guessing the first one is the "correct" one.